src/status_im/ui/screens/intro/views.cljs

```diff
@@ -105,25 +106,26 @@
                                                                  -20
                                                                  (/ view-height 12))}}
    (for [acc multiaccounts]
-     (let [selected? (= (:id acc) selected-id)]
-       ^{:key (:pubkey acc)}
+     (let [selected? (= (:id acc) selected-id)
+           public-key (get-in acc [:derived constants/path-whisper-keyword :publicKey])]
```

Comment:
can't we use public-key instead of publicKey to keep syntax homogeneous?

Reply:
i believe we can, we just need to ask @gravityblast to change this

Reply:
ok not for this PR then we need this tested and merged asap